tests/test_angle.py

```diff
@@ -77,7 +77,7 @@ def test_invalid():
 
     # Invalid value type
     np.testing.assert_raises(TypeError, coord.Angle, theta, coord.degrees)
-    np.testing.assert_raises(ValueError, coord.Angle, 'spam', coord.degrees)
+    np.testing.assert_raises((ValueError, TypeError), coord.Angle, 'spam', coord.degrees)
```

Comment:
```suggestion
    # the jax-galsim version of this class raises a TypeError here
    # instead of a ValueError
    np.testing.assert_raises((ValueError, TypeError), coord.Angle, 'spam', coord.degrees)
```